server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueFactory.java

```diff
@@ -56,6 +57,7 @@
         private final MailQueueView.Factory mailQueueViewFactory;
         private final Clock clock;
         private final MailQueueItemDecoratorFactory decoratorFactory;
+        private final CassandraMailQueueViewConfiguration configuration;
```

Comment:
/!\ This object related to MailQueueView should not be reused here (out of mail queue view)

Please create a new `RabbitMQMailQueueConfiguration` class for handling your additional property.